tests/behat/features/capabilities/group/group-edit-content-in-group.feature

```diff
@@ -34,3 +35,46 @@ Feature: Move content after creation
     # Edit topic
     When I click "Edit content"
     Then I should see "Moving content after creation function has been disabled. In order to move this content, please contact a site manager."
+
+    # Edit topic as SM to move in a new group.
+    When I am logged in as "smith"
+    And I am on "/all-topics"
+    And I should see "I love this sport"
+    And I click "I love this sport"
+    Then I should see "Kayaking"
+    And I click "Edit content"
+    And I select group "Motorboats"
+    And I wait for AJAX to finish
+    And I press "Save"
+    Then I should see "Motorboats"
+
+    When I am logged in as "harry"
+    And I am on the stream of group "Motorboats"
+    Then I should see "harry created a topic in Motorboats"
+    And I should see "I love this sport"
+    And I am on the stream of group "Kayaking"
+    And I should not see "I love this sport"
+
+    # Edit topic as SM to move outside of a group in community.
+    When I am logged in as "smith"
+    And I am on "/all-topics"
+    And I should see "I love this sport"
+    And I click "I love this sport"
+    And I click "Edit content"
+    And I select group "- None -"
+    And I wait for AJAX to finish
+    And I press "Save"
+    Then I should not see "Motorboats" in the "Main content"
+    And I should not see "Kayaking" in the "Main content"
+
+    When I am logged in as "harry"
+    And I am on the stream of group "Motorboats"
+    Then I should not see "harry created a topic in Motorboats"
+    And I should not see "I love this sport"
+    And I am on the stream of group "Kayaking"
+    And I should not see "I love this sport"
+    And I run cron
```

Comment:
@ronaldtebrake why is this? Should we move it maybe towards the first steps? This could have influence on the activity items.